features/step_definitions/individual_steps.rb

```diff
@@ -1291,6 +1291,12 @@ class VlpDocument
 end
 
 Then(/^Individual should see confirmation and continue$/) do
+  find_all('.interaction-click-control-continue')[0].click
+  expect(page).to have_content "Based on the information you entered, you may be eligible to enroll now but there is limited time"
+  find_all('.interaction-click-control-continue')[0].click
+end
+
+Then(/^Individual should see successful sep message$/) do
```

Comment:
Seems to be a duplicate individual step https://github.com/ideacrew/enroll/blob/8182085a20f550e1454bd8525d4b953a9ed39992/features/step_definitions/individual_steps.rb#L1299-L1306